routes/check-if-project-exists.js

```diff
@@ -0,0 +1,81 @@
+var request = require("request");
+
+module.exports = function(config) {
+  return function(req, res) {
+    var publishURL = config.publishURL;
+    var token = config.cryptr.decrypt(req.session.token);
+
+    if(!req.session.user) {
```

Comment:
This should probably be refactored into a piece of middleware and attached to the routes you want it in.